testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -38,7 +38,7 @@
                                        XPDB_small, PSF, DCD, CONECT, CRD,
                                        INC_PDB, PDB_xlserial, ALIGN, ENT,
                                        PDB_cm, PDB_cm_gz, PDB_cm_bz2,
-                                       PDB_mc, PDB_mc_gz, PDB_mc_bz2)
+                                       PDB_mc, PDB_mc_gz, PDB_mc_bz2, PDB_CRYOEM_BOX)
```

Comment:
PEP8: the addition of `PDB_CRYOEM_BOX` to this line is making the number of characters exceed 79. I would add it on a newline.